### PR TITLE
Templates -> Scaffolds Patch Matrix Rename

### PR DESCRIPTION
**Pull request by korgon**
Renaming `template` -> `scaffold` for Snap Templates.

### CODE REVIEW

File: .github/workflows/patchTemplate.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Patch Template
+name: Patch Scaffolds
```
Comment (dkonieczek) [inline .github/workflows/patchTemplate.yml:1]: rename the file `patchTemplate.yml` to `patchScaffold.yml`?

Comment (korgon) [reply .github/workflows/patchTemplate.yml:1]: Good call.

**Comment by korgon** [comment]
This change will need to be lined up with the Snapfu release and template->scaffold rename process.

**Review by dkonieczek — APPROVED** [review]

File: docs/edit-json.md [hunk 2]
```diff
@@ -113,4 +113,24 @@ steps:
...
                 - remove:
                     path: ['searchspring', 'tags', 0]
+```
+
+### `move`
+To move a key or an entry (or rename).
+
+This action does not support `properties` usage.
+
+When using `path`, the location of the property to alter is provided in an array format. If no `newPath` is provided the action will do nothing. If the `newPath` allready exists, the action will do nothing unless a modifier is provided. There are two supported modifiers that can be used when we want to deal with an existing `newPath`: `merge` and `replace`. When `merge` is used, we will attempt to merge the original object with the existing one. When `replace` is used, we will overwrite the existing entry.
```
Comment (dkonieczek) [inline docs/edit-json.md:123]: `allready` -> `already`

**Review by dkonieczek — APPROVED** [review]

**Review by dkonieczek — APPROVED** [review]

**Pull request merged**